index.js

```diff
@@ -8,8 +8,9 @@ module.exports = function markdownItPandoc(md, markdownItPandocOpts) {
                , definition_lists:           true
                , fenced_divs:                true
                , footnotes:                  true
-               , implicit_figures:           true
                , grid_tables:                true
+               , highlightjs:                true
+               , implicit_figures:           true
```

Comment:
Why did `implicit_figures` move, you ask? It was the only option out of alphabetical order, so I figured it was harmless to fix that.